### DISCUSSION TITLE
Handling `unchanged_toast`

### DISCUSSION THREAD

**Discussion by stevenolay** [supabase/supabase#5535] [Questions] [answered]
I am occasionally getting real time updates from supabase/realtime that squash the content of some of my text fields to a string saying `unchanged_toast`. I understand that TOAST is related to the way PSQL handles variable length text and that it should help keep IO down. However, for my use-case, I always need the full content of the objects.  

Is there a way I can force supabase/realtime or PSQL to include the full data of the text field. Is there some setting I need to change? 

Thanks!

Steven Layne

**Comment by w3b6x9** [comment, marked as answer]
@stevenolay are you self-hosting realtime or running a hosted supabase project?

it's not possible to reveal the full contents of `unchanged_toast` directly with realtime. you can always make an api request to query for the data after the fact.

**Comment by stevenolay** [comment]
Hey,

I am self hosting!

It appears if I set the replica identity to full for the table in question,
when there is an unchanged_toast field, the full contents of that field
will be present in the Old Record.

I don’t want to setup a process right now to zip that stuff together, and I
am concerned about IO. So I have resorted to doing a full lookup as you
have suggested.

Thanks!
